src/pod.jl

```diff
@@ -241,3 +241,19 @@ function isk8s()
     end
     return in_kubepod
 end
+
+
+"""
+    az_node_selector!() -> Dict
+
+Add a nodeSelector to a new pod that will ensure that it runs in the same availability zone
+as the manager pod.
+"""
+function az_node_selector!(pod)
+    pod_name = ENV["HOSTNAME"]
+    node_name = readchomp(`$(kubectl()) get pod $pod_name -o jsonpath='{.spec.nodeName}'`)
+    zone = readchomp(`$(kubectl()) get node $node_name -o jsonpath='{.metadata.labels.topology\.kubernetes\.io/zone}'`)
```

Comment:
Should be made into a separate function: `pod_zone`
```suggestion
    node_name = readchomp(`$(kubectl()) get pod $pod_name -o jsonpath='{.spec.nodeName}'`)
    zone = readchomp(`$(kubectl()) get node $node_name -o jsonpath='{.metadata.labels.topology\.kubernetes\.io/zone}'`)
```